src/hearings/containers/view-hearing/hearing-adjourned-summary/hearing-adjourned-summary.component.ts

```diff
@@ -1,30 +1,36 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 import { Store } from '@ngrx/store';
-import { Subscription } from 'rxjs';
+import { Observable, Subscription } from 'rxjs';
 import { filter } from 'rxjs/operators';
+import { LovRefDataModel } from '../../../models/lovRefData.model';
 import { HearingActualsMainModel } from '../../../models/hearingActualsMainModel';
-import { HearingActualsStateData } from '../../../models/hearingActualsStateData.model';
 import * as fromHearingStore from '../../../store';
 
 @Component({
   selector: 'exui-hearing-adjourned-summary',
   templateUrl: './hearing-adjourned-summary.component.html',
 })
 export class HearingAdjournedSummaryComponent implements OnInit, OnDestroy {
-  public hearingActualsModel: HearingActualsMainModel;
+  public hearingState$: Observable<fromHearingStore.State>;
+  public hearingActualsMainModel: HearingActualsMainModel;
+  public adjournReasons: LovRefDataModel[];
   public subscription: Subscription;
 
-  constructor(private readonly store: Store<fromHearingStore.State>) {
+  constructor(private readonly hearingStore: Store<fromHearingStore.State>,
+              private readonly route: ActivatedRoute) {
+    this.adjournReasons = this.route.snapshot.data.adjournReasons;
   }
 
   public ngOnInit(): void {
-    this.subscription = this.store.select(fromHearingStore.getHearingActuals)
+    this.hearingState$ = this.hearingStore.select(fromHearingStore.getHearingsFeatureState)
       .pipe(
-        filter((state: HearingActualsStateData) => !!state.hearingActualsMainModel),
-      )
-      .subscribe((state: HearingActualsStateData) => {
-        this.hearingActualsModel = state.hearingActualsMainModel;
-      });
+        filter(state => !!state.hearingActuals.hearingActualsMainModel),
+    );
+
+    this.subscription = this.hearingState$.subscribe(state => {
+      this.hearingActualsMainModel = state.hearingActuals.hearingActualsMainModel;
```

Comment:
should use the new subscription / observer pattern, i.e. next: etc

Reply:
Thank you @phillip-whitaker-hmcts I have modified the code as per your suggestion.